rust/crates/greenbone-scanner-framework/src/entry/mod.rs

```diff
@@ -208,9 +214,6 @@ impl RequestHandlers {
                             rh.prefix(),
                             rh.path_segments().join("/")
                         );
-                        if req.method() == Method::HEAD {
```

Comment:
There are tests to verify that HEAD only works on known paths. Either remove the test or move the HEAD check back to the loop.